blog/_posts/2022-02-01-scala-3.1.1-released.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.1 and forward compatibility news
+---
+
+Hello from the Scala 3 team! We are back with a new release of the Scala compiler and some exciting news about improvements in forward compatibility.
+
+## Improvements is Scala 3.1.1
+
+Scala 3.1.1 is the patch release focused on bug fixes. Although there are no new additions to the language, you can still spot some noticeable improvements:
+
+- You can use `using` clauses to introduce subtyping relations between types in the function body.
+
+  Now, the code like below should compile. Any value of type `A` can be used where a value of type `B` is expected as long as a given instance of `A <:< B` is in scope:
+
+  ```scala
+    def example[A, B](a: A)(using A <:< B) =
+      val b: B = a
+      ???
+  ```
+
+  To know more see [PR #13662](https://github.com/lampepfl/dotty/pull/13662) and [related issue](https://github.com/lampepfl/dotty/issues/12955).
```

Comment:
s/know/learn/ 

(I would not be able to explain why this is more idiomatic, but it is)